hooks/pre/32-install_selinux_packages.rb

```diff
@@ -12,5 +12,10 @@
   packages << 'candlepin-selinux' if katello_enabled?
   packages << 'pulpcore-selinux' if pulpcore_enabled?
 
+  if katello_enabled? && el8?
+    # candlepin-selinux pulls in candlepin which requires pki-core
+    execute!('dnf module enable pki-core --assumeyes')
```

Comment:
```suggestion
    execute!('dnf module enable pki-core --assumeyes', false, true)
```

This avoids having the following output every time:

```
2021-02-17 20:51:27 [NOTICE] [root] Running validation checks
Executing: dnf module enable pki-core --assumeyes
Last metadata expiration check: 0:10:00 ago on Wed 17 Feb 2021 08:41:28 PM UTC.
Dependencies resolved.
================================================================================
 Package           Architecture     Version             Repository         Size
================================================================================
Enabling module streams:
 pki-core                           10.6                                        pki-deps                           10.6                                       
Transaction Summary
================================================================================

Complete!
```